apps/ledger-live-mobile/src/components/RootNavigator/BaseNavigator.tsx

```diff
@@ -548,8 +548,7 @@ export default function BaseNavigator() {
         name={ScreenName.DeviceConnect}
         component={DeviceConnect}
         options={{
-          title: t("deviceConnect.title"),
-          headerRight: () => null,
+          ...deviceConnectHeaderOptions(t),
```

Comment:
unnecessary spread, also maybe memoize this

Reply:
But `deviceConnectHeaderOptions` is not a component but a function, how can i memoize it with the React API ?

Reply:
Ok never mind ;)